tools/ansible/tasks/logging/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: win
+  gather_facts: no
+  tasks:
+    - name: Ping test
```

Comment:
I'm not sure why a ping test is being done. This would be testing if ansible can reach the host, but thats already being tested by running the playbook itself.

Reply:
This is acceptable to me, and I'd prefer it to an extra step. Ideally the user follows the documentation properly and doesn't reach a state where this happens. And if they do, things still fail in an easily understandable way assuming they are familiar with ansible.

My preference is removing the win_ping, but we can grab someone elses opinion as well if you think it makes sense.

Reply:
My preference would be to remove it too.

Reply:
Sounds good. I'll remove it